sweep/sweeper.go

```diff
@@ -1421,7 +1421,7 @@ func (s *UtxoSweeper) handleNewInput(input *sweepInputMessage) {
 	)
 	if err != nil {
 		err := fmt.Errorf("wait for spend: %w", err)
-		s.signalResult(pi, Result{Err: err})
+		s.markInputFailed(pi, err)
```

Comment:
As is, this can only error out if we fail to obtain the spend notification. At this point, shouldn't we just tear down the entire sweeper to crash and then force a full daemon restart? If we can't get spend notifications, then we don't know when anything is confirmed, which would impact the sweeper's ability to know when it should stop trying to increase the fee.

Reply:
This is a very good point! Will address this in PR 6 to avoid rebase conflicts.